docs/content/intro/quickstart.md

```diff
@@ -2,43 +2,263 @@
+    $env:PATH+=";$env:USERPROFILE\bin"
+    ```
+
+    The script above downloads kind.exe and adds it to your PATH for the current session.
+    Add the following line to your [PowerShell Profile](https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/) to make the change permanent.
+
+    ```powershell
+    $env:PATH+=";$env:USERPROFILE\bin"
+    ```
+
+1. Create a Kubernetes cluster by running the following command:
+    ```
+    kind create cluster
+    ```
+
+1. Verify that you can connect to the cluster using kubectl:
+    ```
+    kubectl cluster-info
+    ```
+
+[Helm]: https://helm.sh/docs/intro/install/
+[Minikube]: https://minikube.sigs.k8s.io/docs/start/
+[KinD]: https://kind.sigs.k8s.io/docs/user/quick-start/
+[kubectl]: https://kubernetes.io/docs/tasks/tools/#kubectl
+[Docker]: https://docs.docker.com/get-docker/
+
+### Install the Brigade CLI
+
+Install the Brigade CLI, brig, by copying the appropriate binary from our releases page into a directory on your machine that is included in your PATH environment variable.
 
-The easiest way to install Brigade into your Kubernetes cluster is to install it using [Helm](https://helm.sh/), the Kubernetes Package Manager.
+**linux**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-linux-amd64
+chmod +x /usr/local/bin/brig
+```
 
+**macos**
 ```bash
-# add Brigade chart repo
-helm repo add brigade https://brigadecore.github.io/charts
-# install Brigade
-helm install -n brigade brigade/brigade
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-darwin-amd64
+chmod +x /usr/local/bin/brig
+```
 
-# if you want to activate Generic Gateway, you should use this command
-# helm install -n brigade brigade/brigade --set genericGateway.enabled=true
+**windows**
+```powershell
+mkdir -force $env:USERPROFILE\bin
+(New-Object Net.WebClient).DownloadFile("https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-windows-amd64.exe", "$ENV:USERPROFILE\bin\brig.exe")
+$env:PATH+=";$env:USERPROFILE\bin"
 ```
 
-You will now have Brigade installed. [Kashti](https://github.com/brigadecore/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
+The script above downloads brig.exe and adds it to your PATH for the current session.
+Add the following line to your [PowerShell Profile](https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/) to make the change permanent.
 
-## Install brig
+```powershell
+$env:PATH+=";$env:USERPROFILE\bin"
+```
 
-Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds, etc. To get `brig`, navigate to the [Releases page](https://github.com/brigadecore/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 1.2.1 version in this way:
+## Install Brigade
```

Comment:
Are the installation instructions here the same as those in `install.md`?
I wonder if it is going to be more difficult to maintain them given they are in two separate files, and for now are the same.

Reply:
They are not exactly the same, these instructions are targeted towards people running a local development cluster that will require port forwarding. See the OP for the reasoning behind having a quickstart that doesn't link you off to separate pages. IMO installation instructions don't change often enough for this to be a problem, it's a short helm install. If y'all hate the idea though, I can redo the quickstart.

Though per your other comment, I am adding alternate instructions for working with a remote development cluster (like a dev aks cluster).